www/src/pages/updates/notes/2019-01-29.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: January 29, 2019
+---
+
+## 🎉 Thumbprint public launch
+
+Today we officially launched a newly redesigned [Thumbprint](https://thumbprint.design). Content has been reorganized and section links are always available from the trusty sidebar.
+
+We also published an [introduction to Thumbprint](https://medium.com/thumbtack-design/an-introduction-to-thumbprint-61f1dfde5c93) on our Thumbtack Design account on Medium. Feel free to share.
+
+Special thanks to Jon K. for help with the redesign and QA.
+
+## 💾 Now open source
+
+Thumbprint is now an officially open source project and our [repository](https://github.com/thumbtack/thumbprint) is public. This brings with it some advantages:
+
+-   Gives back to wider open source community that served as inspiration for us.
+-   Encourage contributions to Thumbprint.
+-   Provides prospective employees a view into our process and technology.
```

Comment:
Thoughts on putting this first and making it more direct?

> Assist with design and engineering recruiting by providing prospective employees a view into our process and technology

Or:

> Assist with design and engineering recruiting by promoting the Thumbtack brand

Reply:
The first one sounds better to me, more concrete.